contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -104,7 +113,23 @@ var ChannelListItem = BaseViews.BaseListEditableItemView.extend({
 		'click .edit_channel':'edit_channel',
 		'click .delete_channel' : 'delete_channel',
 		'click .channel_toggle': 'toggle_channel',
-		'click .save_channel': 'save_channel'
+		'click .save_channel': 'save_channel',
+		'keyup #new_channel_name': 'update_title',
+		'keyup #new_channel_name': 'update_title',
+		'paste #new_channel_name': 'update_title',
+	},
+	update_title:function(event){
```

Comment:
Prevents blank titles: https://trello.com/c/U3bvp4rP/227-editors-should-use-placeholder-text-that-disappears-when-the-user-starts-typing